config/local-argo-deployment.yml

```diff
@@ -655,3 +655,67 @@ data:
   # Fake credentials for localstack that Argo uses so it avoids looking up IAM
   accessKey: c2hoaGhoaGg=
   secretKey: aXRzYXNlY3JldA==
+---
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
It could just be my inability to read kube config, but I don't _think_ this is used, and it's just hard-coded below on line 704?

Reply:
You're right - it's not doing an `envFrom`.